README.MD

```diff
@@ -1,11 +1,11 @@
 BrandedToken.js
 ============
 
-![Build master](https://img.shields.io/travis/OpenSTFoundation/brandedtoken.js/master.svg?label=build%20master&style=flat)
-![Build develop](https://img.shields.io/travis/OpenSTFoundation/brandedtoken.js/develop.svg?label=build%20develop&style=flat)
-![npm version](https://img.shields.io/npm/v/@openstfoundation/brandedtoken.js.svg?style=flat)
+![Build master](https://img.shields.io/travis/OpenST/brandedtoken.js/master.svg?label=build%20master&style=flat)
+![Build develop](https://img.shields.io/travis/OpenST/brandedtoken.js/develop.svg?label=build%20develop&style=flat)
+![npm version](https://img.shields.io/npm/v/@openst/brandedtoken.js.svg?style=flat)
 ![Discuss on Discourse](https://img.shields.io/discourse/https/discuss.openst.org/topics.svg?style=flat)
-![Chat on Gitter](https://img.shields.io/gitter/room/OpenSTFoundation/SimpleToken.svg?style=flat)
+![Chat on Gitter](https://img.shields.io/gitter/room/OpenST/SimpleToken.svg?style=flat)
```

Comment:
the gitter links should not be updated (they would break)

Reply:
😅 the badge doesnt even link to the gitter channel... so we ll fix this in a patch version later; good for now